hnn_core/network.py

```diff
@@ -201,8 +344,6 @@ def __init__(self, params, add_drives_from_params=False,
         self.n_cells = sum(len(self.pos_dict[src]) for src in
                            self.cell_types)
 
-        self._set_default_connections()
-
```

Comment:
in a future PR, `add_drives_from_params` should be moved to `default_network` function